context/app/static/js/components/Detail/ProvTableTile/ProvTableTile.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useContext } from 'react';
+import React, { useContext, useMemo } from 'react';
```

Comment:
I wouldn't use memo... I don't think we're doing any re-rendering in a tight loop, so it's a bit of extra complexity for us to worry about, without gaining much.

Reply:
I needed to use `useMemo` here, `useEffect` was causing it to get caught re-rendering.

Reply:
Needed may be the wrong word. `useMemo` solved the issue and I figured it would be better to use it than seek some other solution since this will be removed once we get `descendant_counts` from the api.